gui/default/assets/lang/lang-en.json

```diff
@@ -33,6 +33,7 @@
     "Anonymous usage report format has changed. Would you like to move to the new format?": "Anonymous usage report format has changed. Would you like to move to the new format?",
     "Applied to LAN": "Applied to LAN",
     "Apply": "Apply",
+    "Version Deleted Only":"Version Deleted Only",
```

Comment:
Did you insert this string manually?  Because it sure looks like having a different white-space pattern.

The correct way to update this file is this:

    go run build.go translate

Possibly after first emptying the file (just `{}` as content), in case you removed strings.